roles/basic_prog_pkgs/vars/main.yml

```diff
@@ -1,18 +1,13 @@
 ---
 # vars file for basic-prog-pkgs
 basic_prog_pkgs_intro_development:
- - artha
- - bless
- - chromium-browser
- - fonts-crosextra-caladea
- - fonts-crosextra-carlito
- - geany
- - idle
  - libreoffice
- - libreoffice-templates
- - libreoffice-style-tango
  - logisim
  - meld
+ - sqlite
  - sqlitebrowser
  - thonny
- - wireshark
+
+basic_prog_pkgs_pip:
+ - flake8-docstrings
+ - darglint
```

Comment:
At least the former is available as `python3-flake8-docstrings` in Ubuntu. Not sure about `darglint`. But it might be nice to use the system Python in this course?

Reply:
I don't see darglint, but Thonny does appear to recognize system packages in its "Package Manager" so I can split this.